graylog2-web-interface/src/components/common/EntityFilters/EntityFilters.tsx

```diff
@@ -68,13 +76,27 @@ const EntityFilters = ({ attributes = [], filterValueRenderers, urlQueryFilters,
   }, [onChangeFiltersWithTitle, setUrlQueryFilters]);
 
   const onCreateFilter = useCallback((attributeId: string, filter: Filter) => {
+    sendTelemetry(TELEMETRY_EVENT_TYPE.ENTITY_DATA_TABLE.FILTER_CREATED, {
+      app_pathname: getPathnameWithoutId(pathname),
+      app_section: appSection,
+      app_action_value: 'new-filter-created',
```

Comment:
The other `app_action_value` values start with `filter-`, maybe `filter-created` is sufficient? This way we would keep the naming similar.